### ISSUE TITLE
Wrong feature locations after sticky-end ligation

### ISSUE THREAD

**Issue by jan-glx** [pydna-group/pydna#88]
The new feature locations computed when adding two  `DSeqrecords` with sticky ends are off:

```python
from pydna.genbank import Genbank
from Bio.Restriction import BamHI, NdeI
```

```python
gb = Genbank("bjornjobb@gmail.com")
puc19 = gb.nucleotide("M77789.2")
bb, insert = puc19.cut(NdeI,BamHI)
```

```python
puc19_ = (bb + insert).looped().synced(puc19)
```

```python
print(puc19_.extract_feature(2), "\n")
print(puc19.extract_feature(6))
```

    Dseqrecord
    circular: False
    size: 137
    ID: pBR322
    Name: pBR322
    Description: description_rc
    Number of features: 1
    /molecule_type=DNA
    Dseq(-137)
    TTCG..TTAA
    AAGC..AATT 
    
    Dseqrecord
    circular: False
    size: 137
    ID: pBR322
    Name: pBR322
    Description: Cloning vector pUC19, complete sequence_rc
    Number of features: 1
    /molecule_type=DNA
    Dseq(-137)
    TCGC..GGGC
    AGCG..CCCG

```python
print(puc19_.features[2], "\n")
print(puc19.features[6], "\n")
```

    type: misc_feature
    location: [539:676](-)
    qualifiers:
        Key: note, Value: ['pBR322']
     
    
    type: misc_feature
    location: [547:684](-)
    qualifiers:
        Key: note, Value: ['pBR322']

**Comment by BjornFJohansson** [comment]
Thanks! Ill have a look at this.

**Comment by BjornFJohansson** [comment]
Still working on this, regular work cam in the way.

**Comment by BjornFJohansson** [comment]
Have a solution for this, look out for a new release.

**Comment by BjornFJohansson** [comment]
Solved in 5.2.0 I hope. Feel free to reopen if not.
